pkg/netconf/frr.go

```diff
@@ -145,6 +173,7 @@ func assembleVRFs(kb config) []VRF {
 			ImportVRFNames: i.ImportVRFs,
 			IPPrefixLists:  i.prefixLists(),
 			RouteMaps:      i.routeMaps(),
+			EnforceFirstAS: enforceFirstASString,
```

Comment:
I think this makes it harder to read the template. Why not passing a boolean flag and then write the actual string into the template?

Reply:
done